src/rfc3986/parseresult.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 """Module containing the urlparse compatibility logic."""
 from collections import namedtuple
+from typing import Optional
```

Comment:
Please don't polute the namespace. Import typing as t instead

Reply:
I can't, current lint rule forbide `import typing as t`, 

`https://github.com/asottile/reorder_python_imports` want 
```
import typing as t
from collections import namedtuple
```

and flake8 want 

```
from collections import namedtuple
import typing as t
```

they are fighting in the pre-commit hooks.

Reply:
I'll just remove flake8-import-order